lib/event-source.js

```diff
@@ -0,0 +1,79 @@
+var mongojs = require('mongojs');
+var inflect = require('i')();
+var _ = require('lodash');
+var EventSource = function() {
+};
+
+EventSource.prototype.init = function(router, adapter, options) {
+  this.router = router;
+  this.adapter = adapter;
+  this.options = options;
+  this.db = mongojs((process.env.OPLOG_MONGODB_URL || process.argv[3]) + '?slaveOk=true');
+}
+
+EventSource.prototype.initRoute = function(routeName) {
+  this.router.get(routeName + '/changes', this.handler.bind(this));
+};
+
+EventSource.prototype.handler = function(req, res) {
+    var me = this;
+    var routeName = /\/(.+)\/changes/.exec(req.route.path)[1];
+    var query = {
+      ns : this.options.db + '.' + routeName,
+    };
+    
+    if (req.query.event) query.op = req.query.event;
+    if (req.query.seq) {
+      var operation = req.query.seq.split('=');
+      //stop this block if we're being sent bad ops
+      if (_.contains(['gt', 'lt', 'gte', 'lte'], operation[0])) { 
+        query['ts'] = {};
+        //this will result in something like id : {$gt : 12345}
+        query['ts']['$' + operation[0]] =  mongojs.Timestamp.fromNumber(parseInt(operation[1], 10), 2);
+      }
+    }
+    var coll = this.db.collection('oplog.rs');
+    var options = {
+      tailable : true,
+      awaitData : true
+    };
+    
+    this.writeSSEHead(req, res, function() {
+      var stream = coll.find(query).sort({$natural : -1}).limit(parseInt(req.query.limit, 10));
```

Comment:
was expecting that without a Last-Event-Id we would locate the last entry with paging = 1
and that that entry would subsequently be used to initiate the stream 
like this : https://github.com/cayasso/mongo-oplog/blob/master/lib/index.js#L104-L126

have you tested this with multiple change events ?